ha/core/event_analyzer/watcher/watcher.py

```diff
@@ -51,6 +51,7 @@ def __init__(self, consumer_id: int, message_type: str, consumer_group: str,
         self.parser = event_parser
         self.subscriber = subscriber
         self._validate()
+        # TODO : Initialize MessageBus.init() reference EOS-26999
```

Comment:
Please remove this TODO from here and add it in event_analyzerd file. Currently we are not using this watcher file.

Reply:
@akash2144  please update the TODO as " TBD: call MessageBus.init() from here if this watcher needs to be used in HA. Ref  EOS-26999" 
@Madhura-08  TODO is added since we are not doing init here (since this is unused code); so tomorrow if this watcher is used, developer will know that init needs to be done here

Reply:
Done

Reply:
Oh Ok.. Got it. Thanks!